inference-engine/include/cpp/ie_cnn_network.h

```diff
@@ -54,6 +54,8 @@ class INFERENCE_ENGINE_API_CLASS(CNNNetwork) {
      * @param network Pointer to the ngraph::Function object
      */
     explicit CNNNetwork(const std::shared_ptr<const ngraph::Function>& network);
+    explicit CNNNetwork(const std::shared_ptr<ngraph::Function>& network);
+    explicit CNNNetwork(const ICNNNetwork& network);
```

Comment:
is it suppose that non-const object can be touched and transformed? can we write such docs in hpp?

We need explicit docs for all 3 ctors.

Reply:
sure I will document these ctors. It is a PoC